src/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+
+import './index.css';
+import { numberLiteralTypeAnnotation } from '@babel/types';
+
+let linesPerPage = 10;
+
+class App extends React.Component{
+  constructor(props,context){
+    super(props,context)
+    this.state = {
+      data :  []
+    }
+  }
+
+  getBlockInfo = () => {
+    axios.get(`${"http://localhost:3001/list"}/${linesPerPage}`).then(res=>{

```

Comment:
I think you can use `async/await` syntax here. Can you try?

Reply:
resolved